python-checks/src/main/java/org/sonar/python/checks/UnusedLocalVariableCheck.java

```diff
@@ -56,6 +65,16 @@ private static boolean isCallingLocalsFunction(AstNode functionTree) {
       .anyMatch(node -> "locals".equals(node.getTokenValue()));
   }
 
+  private Set<String> extractStringInterpolationWords(AstNode functionTree) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "extractStringInterpolationWords" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-243&issues=AWDbNsMumexALlUbKL5Z&open=AWDbNsMumexALlUbKL5Z)

[ISSUE_KEY]: AWDbNsMumexALlUbKL5Z